priv/yokozuna.schema

```diff
@@ -103,6 +103,46 @@
   end
 }.
 
+%% @doc Whether the throttle for Yokozuna active anti-entropy is enabled.
+{mapping, "search.anti_entropy.throttle", "yokozuna.aae_throttle_kill_switch", [
+  {default, on},
+  {datatype, {flag, off, on}},
```

Comment:
Need to switch the On/Off defaults when we switch the `kill_switch` to `enabled` - I'm assuming we're mapping `search.anti_entropy.throttle" value of `on` to `yokozuna.aae_throttle_kill_switch` of `false` by the order of the `{flag, off, on}` tuple above, but not sure. Please confirm and make necessary changes.